.pipelines/.vsts-vhd-builder-release-windows.yaml

```diff
@@ -19,7 +19,7 @@ pr:
     - vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh
     - vhdbuilder/packer/generate-vhd-publishing-info.sh
     - vhdbuilder/packer/init-variables.sh
-    - vhdbuilder/packer/windows-image.env
+    - vhdbuilder/packer/windows-image.env.sh
```

Comment:
Do you plan to release new AKS Windows images with non-security changes?

Reply:
The release is a separate process - but I do want to build them automatically.

There's also the need to set registry keys in the VHDs to enable any fixes for AKS. That'll have to be manual, sadly.